static/js/util/util.js

```diff
@@ -365,3 +370,23 @@ export function createForm(url: string, payload: CheckoutPayload): HTMLFormEleme
 export function formatPrice(price: string): string {
   return `$${price}`;
 }
+
+/**
+ * Returns total courses and passed courses in program.
+ */
+export function programCourseInfo(program: Program): Object {
+  let totalCourses = 0;
+  let totalPassedCourses = 0;
+  let passedCourses: Course;
+
+  if (program.courses) {
+    totalCourses = program.courses.length;
+    passedCourses = program.courses.find(course => course.status === STATUS_PASSED);
+    totalPassedCourses = passedCourses ? passedCourses.length : 0;
```

Comment:
`course.status` was removed recently, there is only status at the run level now. You will need to check  `course.runs[0].status` instead